### PR TITLE
fix: refund excess tokens

### PR DESCRIPTION
**Pull request by fedealconada**
fixes https://github.com/sherlock-audit/2023-03-sense-judging/issues/29

### CODE REVIEW

File: src/RollerPeriphery.sol [hunk 1]
```diff
@@ -328,11 +334,8 @@ contract RollerPeriphery is Trust {
             );
     }
 
-    function _transferRemainingUnderlying(AutoRoller roller, address receiver) internal {
-        // transfer any remaining underlying to receiver
-        ERC20 underlying = ERC20(roller.adapter().underlying());
-        uint256 remaining = underlying.balanceOf(address(this));
-        if (remaining > 0) underlying.safeTransfer(receiver, remaining);
+    function _refundExcess(AutoRoller roller, ERC20 token, address receiver, uint256 amt) internal {
+        if (amt > 0) address(token) == ETH ? payable(receiver).transfer(amt) : token.safeTransfer(receiver, amt);
```
Comment (IAm0x52) [inline src/RollerPeriphery.sol:338]: Should be call rather than transfer

**Review by IAm0x52 — APPROVED** [review]
Fixes look good. Excess sell tokens are now returned inside _toTarget and _fromTarget

File: src/RollerPeriphery.sol (hunk 1 above)
Comment (fedealconada) [reply src/RollerPeriphery.sol]: fixed, take a look please :)

**Review by IAm0x52 — APPROVED** [review]
Fixes look good. Both occurrences of transfer have been addressed.

**Pull request merged**